Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -479,8 +479,19 @@ void RomBrowserWidget::customContextMenuRequested(QPoint position)
         this->action_PlayGameWith->setText("Play Game with Disk");
     }
 
+    if (view == this->listViewWidget)
+    { // list view
+        this->categorySubmenu = this->mainContextMenu->addMenu(tr("&Show/Hide Columns"));
+        RomBrowserWidget::on_listViewWidget_columnVisibilityMenuRequested(categorySubmenu);
```

Comment:
Please just use `this->functionName()` instead of the odd `classname::functionName();`

Reply:
Done, will include in next commit. (Don't know how this one slipped by me.)